ci-operator/config/opendatahub-io/distributed-workloads/opendatahub-io-distributed-workloads-main.yaml

```diff
@@ -0,0 +1,42 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-odh-dw
+  steps:
+    cluster_profile: aws
+    test:
+    - as: e2e-odh-dw
+      cli: latest
+      commands: |
+        make install-opendatahub-operator
+        make install-codeflare-operator
+        sleep 10
+        make deploy-codeflare
+        sleep 120
+        cd tests && GOFLAGS=-mod=mod go test -v ./integration -run TestRayCluster
+      from: src
+      resources:
+        requests:
+          cpu: 2000m
+          memory: 3Gi
+    workflow: ipi-aws
```

Comment:
Can we switch this over to using hypershift? https://github.com/openshift/release/pull/39476/files might be able to provide some guidance.

Reply:
I can try it
Anyway this PR is still WIP, problem is that when I mark it as WIP then CI won't run rehearse